docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1073 @@
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.RecognitionResultUpdated += recognitionResultUpdated;
+
+            /// Unset the callback
+            VoiceControlManagerClient.RecognitionResultUpdated -= recognitionResultUpdated;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Sets the recognized commands selection callback invoked when voice command is recognized to voice control manager.
+
+    You can selects valid command results. If you return true. release command to client.
+
+    > **Note**   
+    > If multiple commands are matched, selects valid results in this callback.
+    ```
+    /// callback
+    IEnumerable<VoiceCommand> recognizedCommandsSelection(IEnumerable<VoiceCommand> commands, string recognizedText, string message)
+    {
+        /// Your code
+        return null;    // release commads.
+
+        /// Your code
+        return valid_commands;  // selects valid results.
+    }
+
+    void SetUnsetRecognizedCommandsSelectionCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.SetRecognizedCommandsSelectionDelegate(recognizedCommandsSelection);
+
+            /// Unset the callback
+            VoiceControlManagerClient.SetRecognizedCommandsSelectionDelegate(null);
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Sets a callback function to be called when engine sends audio formats necessary for playing TTS feedback.
+    ```
+    /// callback
+    void feedbackAudioFormatChanged(object sender, FeedbackAudioFormatEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetFeedbackAudioFormatChangedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.FeedbackAudioFormatChanged += feedbackAudioFormatChanged;
+
+            /// Unset the callback
+            VoiceControlManagerClient.FeedbackAudioFormatChanged -= feedbackAudioFormatChanged;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Sets a callback function to be called when engine sends audio streaming for TTS feedback.
+    ```
+    /// callback
+    void feedbackStreaming(object sender, FeedbackStreamingEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetFeedbackStreamingCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.FeedbackStreaming += feedbackStreaming;
+
+            /// Unset the callback
+            VoiceControlManagerClient.FeedbackStreaming -= feedbackStreaming;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Sets TTS streaming callback function is called when the vc client sends audio streaming for TTS feedback.
```

Comment:
There are 2 types of client on voice control framework. The first one is voice control client for normal application. These applications can register their own voice control command such as 'play, stop and pause'. And second one is voice control manager client which can be used on platform application. This application can exist as for only one in the device. And it can start speech recognition and stop. And it can also register its own voice control command including system command such as 'power off, volume up/down'. 

And we will change the sentence as following for more proper explanation. 
```
Sets TTS streaming callback function is called when the voice control engine sends audio streaming for TTS feedback. This audio streaming is requested by the application which uses voice control client api. 
```